ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -548,7 +567,32 @@ export class ObjectDetailComponent implements OnInit {
         },
         maxWidth: '500px',
       })
-    } else {
+    } else if (this.checkIfCcColumn(colId)) {
```

Comment:
If something is a PK and has a Check constraint we will never enter this block

Reply:
addressed the comment